metrics.ini

```diff
@@ -1582,12 +1582,14 @@ TimerSeconds=SL.Global.MenuTimer.ScreenSelectMusicCasual
 PrevScreen=Branch.SSMCancel()
 NextScreen=Branch.AfterSelectMusic()
 
-CodeNames="SortList,SortList2,SortList3,SortList4,EscapeFromEventMode,TogglePatternInfo"
+CodeNames="SortList,SortList2,SortList3,SortList4,EscapeFromEventMode,TogglePatternInfo,Favorite1,Favorite2"
 CodeSortList="@Select-Start"
 CodeSortList2=(PREFSMAN:GetPreference("OnlyDedicatedMenuButtons") or GAMESTATE:GetCurrentGame():GetName() == "pump") and "MenuLeft-MenuRight"
 CodeSortList3=GAMESTATE:GetCurrentGame():GetName() == "pump" and "DownLeft-DownRight"
 CodeSortList4=not PREFSMAN:GetPreference("OnlyDedicatedMenuButtons") and "Left-Right"
 CodeTogglePatternInfo="Select,Select"
+CodeFavorite1=GAMESTATE:GetCurrentGame():GetName() == "pump" and "DownLeft,UpLeft,Center,UpRight,DownRight,UpRight,Center,UpLeft,DownLeft" or "Right,Down,Left,Up,Right"

```

Comment:
Are these codes shown anywhere on-screen? If not, I would like to at least also have the option in the sort menu.